### DISCUSSION TITLE
How to Implement Hover CSS on elements in Custom CSS?

### DISCUSSION THREAD

**Discussion by CosmasSC128** [Budibase/budibase#5006] [Help] [answered]
Is there any way to do this at the moment?

**Comment by mjashanks** [comment]
Not possible at the moment unfortunately. We might be looking into this and other styling capabilities in the next few months, but don't know the details yet.

**Reply by NicolasFabre81** [reply]
Still not ?

**Reply by tw02d008** [reply]
One year later is there any update on the hover? Is it already possible somehow with custom css?

**Comment by mikesealey** [comment, marked as answer]
@tw02d008 

This can be done with an Embed component, and some clever use of your browser's developer tools.

In your published app, using the developer tools, find the element you wish to apply the on-hover rules to, and take the classes...

<img width="1415" alt="image" src="https://github.com/user-attachments/assets/e9d297ae-bf7e-4758-bd09-2aeb6aa87a63" />

You can then use one of those classes to target your element with a :hover rule, inside some `<style></style>` tags

For good measure, in the screenshot below I brought _all_ classes and pasted them into a comment. There may be some trial-and-error involved

<img width="1422" alt="image" src="https://github.com/user-attachments/assets/27e020aa-23c6-4a66-96df-978dd10a95d7" />

Here's an app-export where I sketched this one out, please feel free to take a closer look.
[CSS On Hover-export-1740569902219.tar.gz](https://github.com/user-attachments/files/18983155/CSS.On.Hover-export-1740569902219.tar.gz)

If you're not sure how to import an app, take a look at [our documentation](https://docs.budibase.com/docs/export-and-import-apps#import-an-application).

**Reply by tw02d008** [reply]
Thank you very much. Got it and it is working great. Helps a lot for my fake tables (Repeater with container and paragraphs)
